src/globals.c

```diff
@@ -551,43 +556,36 @@ static bool global_parse_opt(int opt, char *optarg)
 			set_json_format(true);
 		}
```

Comment:
Maybe you should update options "j", "FLAG_NO_PROGRESS" and "FLAG_WAIT_FOR_SCRIPTS" so their if is consistent with the ones below. For example:
`wait_for_scripts = optarg ? strtobool(optarg) : true;`

Reply:
well, I was unsure what was the best way to do it to be honest. hahaha.
So I didn't change what was already in place. I'm thinking that maybe it's useful to create a opt_to_bool() that returns true on NULL or return strtobool(optarg). What do you think?

Reply:
haha, also I had suggested the same thing in my previous review but ended up deleting the comment, but I do think is a good idea. Or since this is only used in globals.c even a #define could work.

Reply:
hahaha. nice. I'll create a define in another commit